web-app/src/app/services/feeds/utils.ts

```diff
@@ -45,3 +48,32 @@ export type AllDatasetType =
   | paths['/v1/gtfs_feeds/{id}/datasets']['get']['responses'][200]['content']['application/json']
   | paths['/v1/datasets/gtfs/{id}']['get']['responses'][200]['content']['application/json']
   | undefined;
+
+export function getLocationName(locations: FeedLocation | undefined): string {
```

Comment:
Why not use the Locations type provided by the API models? I find it confusing to use singular FeedLocation for an array representation. https://github.com/MobilityData/mobility-feed-api/blob/b8dea9e4a7e1444f4849b703cb54dde6e7670623/web-app/src/app/services/feeds/types.ts#L327

Reply:
Maybe I'm not understanding it but `FeedLocation` is set from the auto generated code `components['schemas']['Locations'];` maybe I should call it `FeedLocations` since there could be many locations

Reply:
> Maybe I'm not understanding it but FeedLocation is set from the auto generated code components['schemas']['Locations']

I understand; I would like to know why there is a need to create a new type pointing to the same definition. In the hypothetical future that we add Location(s) to a different entity like Provider(that doesn't exist yet), FeedLocation(s) will be a confusing name, or we will need to create another type.

> maybe I should call it FeedLocations since there could be many locations

Yes, FeedLocations is a better name.

Reply:
Make sense, _EntityLocations_ is a great alias name. Thanks!